paper/numbering.tex

```diff
@@ -4,24 +4,33 @@ \section{Numbering system}\label{numbering-app}
+To define the ordering we use on equational laws, we first conside the case where there is a single indeterminate $\ast$.
+We place a well-ordering on words $w,w'$ with a single indeterminate $\ast$ by declaring $w > w'$ if one of the following holds:
 \begin{itemize}
     \item $w$ has a larger order than $w'$.
     \item $w = w_1 \op w_2$ and $w' = w'_1 \op w'_2$ have the same order $n \geq 1$ with $w_1 > w'_1$.
     \item $w = w_1 \op w_2$ and $w' = w'_1 \op w'_2$ have the same order $n \geq 1$ with $w_1 = w'_1$ and $w_2 > w'_2$.
 \end{itemize}
 Thus, for instance
-$$ 0 < 1 < 0 \op 0 < 0 \op 1 < 1 \op 0 $$
-and
-$$ 1 \op 1 < 0 \op (0 \op 0) < (0 \op 0) \op 0.$$
+$$ \ast < \ast \op \ast < \ast \op (\ast \op \ast) < (\ast \op \ast) \op \ast.$$
 
-We similarly place a well-ordering on equational laws $w_1 \formaleq w_2$ by declaring $w_1 \formaleq w_2 > w'_1 \formaleq w'_2$ if one of the following holds:
+We similarly place a well-ordering on equational laws $w_1 \formaleq w_2$ with a single indeterminate $\ast$ by declaring $w_1 \formaleq w_2 > w'_1 \formaleq w'_2$ if one of the following holds:
 as follows:
 \begin{itemize}
 \item  $w_1 \formaleq w_2$ has a longer order than $w'_1 \formaleq w'_2$.
 \item If $w_1 \formaleq w_2$ has the same order as $w'_1 \formaleq w'_2$, and $w_1 > w'_1$.
 \item If $w_1 \formaleq w_2$ has the same order as $w'_1 \formaleq w'_2$, $w_1 = w'_1$, and $w_2 > w'_2$.
 \end{itemize}
+Thus for instance
+$$ (\ast \op \ast \formaleq \ast \op (\ast \op \ast)) < (\ast \op \ast \formaleq (\ast \op \ast) \op \ast).$$
+
+Finally for equational laws with alphabet $x,y,z,w,u,v,r,s,t$, define the \emph{shape} of that law to be the law formed by replacing all indeterminates with $\ast$; for instance, the shape of \eqref{eq4512}, $(\ast \op \ast) \op \ast \formaleq \ast \op (\ast \op \ast)$, is $(\ast \op \ast) \op \ast \formaleq \ast \op (\ast \op \ast)$.  We then place a well-ordering $w_1 \formaleq w_2$ with indeterminates $x,y,z,w,u,v,r,s,t$ by declaring $w_1 \formaleq w_2 > w'_1 \formaleq w'_2$ if one of the following holds:
+\begin{itemize}
+\item The shape of $w_1 \formaleq w_2$ is greater than the shape of $w'_1 \formaleq w'_2$.
+\item $w_1 \formaleq w_2$ and $w'_1 \formaleq w'_2$ have the same shape, and the string of variables appearing in $w_1 \formaleq w_2$ is lower in the lexicographical ordering (using $x < y < z < w < u < v < r < s < t$) than the corresponding string for $w'_1 \formaleq w'_2$.
+\end{itemize}
+Thus for instance any law of shape $\ast \op \ast \formaleq \ast \op (\ast \op \ast)$ is lower than any law of shape
+$\ast \op \ast \formaleq (\ast \op \ast) \op \ast$.  Among the laws of shape $\ast \op \ast \formaleq \ast \op (\ast \op \ast)$, the lowest is $\x \op \x \formaleq \x \op (\x \op \x)$, which is less than (say) $\x \op \x \formaleq \y \op (\y \op \y)$, which is in turn less than $\x \op \y \formaleq \x \op (\x \op \x)$.
```

Comment:
Most of the ordering is stated in terms of the "greater than" relation, so "lower in the lexicographic ordering" should be "greater in the lexicographic ordering" in the list above.  Alternatively phrase everything in terms of less then, which feels more natural to me.